test/unit/modules/multisignatures.js

```diff
@@ -911,4 +968,252 @@ describe('multisignatures', () => {
 			});
 		});
 	});
+
+	describe('getGroup', () => {
+		beforeEach(done => {
+			stubs.logic.account.getMultiSignature = sinonSandbox
+				.stub()
+				.callsFake((filters, cb) => cb(null, validAccount));
+
+			stubs.getMemberPublicKeys = sinonSandbox
+				.stub()
+				.callsFake(() => Promise.resolve([]));
+
+			stubs.modules.accounts.getAccounts = sinonSandbox
+				.stub()
+				.callsFake((param1, param2, cb) => cb(null, []));
+
+			stubs.modules.accounts.generateAddressByPublicKey = sinonSandbox
+				.stub()
+				.withArgs('key1')
+				.returns('address1');
+
+			library.logic.account.getMultiSignature =
+				stubs.logic.account.getMultiSignature;
+			library.db.multisignatures.getMemberPublicKeys =
+				stubs.getMemberPublicKeys;
+			get('modules').accounts.getAccounts = stubs.modules.accounts.getAccounts;
+			get('modules').accounts.generateAddressByPublicKey =
+				stubs.modules.accounts.generateAddressByPublicKey;
+			done();
+		});
+
+		it('should accept address as parameter', done => {
+			self.getGroup(validAccount.address, (err, scopeGroup) => {
+				expect(library.logic.account.getMultiSignature).to.be.calledWith({
+					address: validAccount.address,
+				});
+				expect(scopeGroup.address).to.equal(validAccount.address);
+				done();
+			});
+		});
+
+		it('should fail if getMultiSignature function returns an error', done => {
+			library.logic.account.getMultiSignature = sinonSandbox
+				.stub()
+				.callsFake((filters, cb) => {
+					cb('Err', null);
+				});
+			self.getGroup(validAccount.address, (err, scopeGroup) => {
+				expect(err).to.equal('Err');
+				expect(scopeGroup).to.not.exist;
+				done();
+			});
+		});
+
+		it('should fail if given address does not return an account', done => {
```

Comment:
wrong description, you're not testing the behavior of `getMultiSignature`, so you cannot be sure of the reason why this function returning empty account